bot_commands.py

```diff
@@ -11,8 +11,9 @@
 
 
 class BotCommands:
-    def __init__(self, send_message):
+    def __init__(self, send_message, metrics_handler):
         self._send_message = step(send_message)
+        self._metrics_handler = metrics_handler
```

Comment:
Why?

Reply:
When I tried testing metrics about a month ago, it wasn't working. I noticed there wasn't a `metrics_handler` being passed in, so I added it in. Do you want me to remove it?